xcresultparser/main.swift

```diff
@@ -21,9 +21,15 @@ struct xcresultparser: ParsableCommand {
     @Option(name: .shortAndLong, help: "The name of the project root. If present paths and urls are relative to the specified directory.")
     var projectRoot: String?
     
+    @Option(name: .shortAndLong, help: "Specify which targets to calculate coverage from")
+    var targets: [String] = []
+    
     @Flag(name: .shortAndLong, help: "Whether to print coverage data.")
     var coverage: Int
     
+    @Flag(name: .shortAndLong, help: "Whether to print test results.")
+    var notestresult: Int
```

Comment:
likewise we should name this 'testResults' to match what the above 'coverage' means. But here we'd have a default of 1 in order to not break older versions.
```
@Flag(name: .shortAndLong, help: "Whether to print test results.")
     var testResults = true
```